cmd/lotus/daemon.go

```diff
@@ -160,6 +166,19 @@ var DaemonCmd = &cli.Command{
 			Name:  "restore-config",
 			Usage: "config file to use when restoring from backup",
 		},
+		&cli.BoolFlag{
+			Name:  "slash-consensus",
+			Usage: "Report consensus fault",
+			Value: false,
+		},
+		&cli.StringFlag{
+			Name:  "from",
+			Usage: "optionally specify the account to report consensus from",
+		},
+		&cli.StringFlag{
+			Name:  "extra",
```

Comment:
I don't understand how we could meaningfully use this `extra` param?

Reply:
I have referred to the official implementation and now removed this parameter